single_subject/batch_single_subject.sh

```diff
@@ -392,15 +396,62 @@ sct_warp_template -d fmri_moco_mean.nii.gz -w warp_template2fmri.nii.gz -a 0 -qc
 
 cd ../t1
 # Segment T1-weighted image (to be used in later steps)
-sct_deepseg_sc -i t1.nii.gz -c t1
+sct_deepseg -task seg_sc_contrast_agnostic -i t1.nii.gz
 
 # Smooth spinal cord along centerline (extracted from the segmentation)
 sct_smooth_spinalcord -i t1.nii.gz -s t1_seg.nii.gz
 # Tips: use flag "-sigma" to specify smoothing kernel size (in mm)
 
+# Second-pass segmentation using the smoothed anatomical image
+sct_deepseg_sc -i t1_smooth.nii.gz -c t1 -qc ~/qc_singleSubj
+
 # Align the spinal cord in the right-left direction using slice-wise translations.
 sct_flatten_sagittal -i t1.nii.gz -s t1_seg.nii.gz
 # Note: Use for visualization purposes only
 
+
+
+# New features (SCT v6.5, December 2024)
+# ======================================================================================================================
+
+# Lesion analysis for SCI and MS lesions
```

Comment:
After moving the lesion segmentation to "New features", we need to remove the original commands: 
https://github.com/spinalcordtoolbox/sct_tutorial_data/blob/3ecf3147cc7521389cb9b536ac79899470811c73/single_subject/batch_single_subject.sh#L118-L132